### PR TITLE
Support COLUMN_FORMAT syntax.

### PR DESCRIPTION
**Pull request by doggeral**
### What problem does this PR solve? 
To support the COLUMN_FORMAT syntex. 
https://github.com/pingcap/parser/issues/476

### What is changed and how it works?
Add the COLUMN_FORMAT support.

### Check List 

 - Unit test

### CODE REVIEW

File: parser.y [hunk 1]
```diff
@@ -2100,6 +2101,20 @@ ColumnOption:
...
+|
+    "COLUMN_FORMAT" "FIXED"
+	{
+		$$ = &ast.ColumnOption{Tp: ast.ColumnOptionColumnFormat, StrValue: "FIXED"}
+	}
+|
+    "COLUMN_FORMAT" "DYNAMIC"
+	{
+		$$ = &ast.ColumnOption{Tp: ast.ColumnOptionColumnFormat, StrValue: "DYNAMIC"}
+	}
```
Comment (tangenta) [inline parser.y]: It would be better to abstract `DEFAULT`, `FIXED`, `DYNAMIC` as a new production named `columnFormat`.

File: parser.y [hunk 2]
```diff
@@ -299,6 +299,7 @@ import (
 	client		"CLIENT"
 	coalesce	"COALESCE"
 	collation	"COLLATION"
+	columnFormat "COLUMN_FORMAT"
```
Comment (tangenta) [inline parser.y]: Please indent the line with tab.

Comment (doggeral) [reply parser.y]: Will do.

File: parser.y (hunk 1 above)
Comment (doggeral) [reply parser.y]: What's the benefit to create new columnFormat name? Can it be reused by anywhere else?

Comment (tangenta) [reply parser.y]: No, it can't. Just in line with the existing style.

Comment (doggeral) [reply parser.y]: got you, will do.

**Comment by doggeral** [comment]
Resolved the conflict.

**Pull request merged**